angular/src/lib/editable-textfield/editable-textfield.component.ts

```diff
@@ -13,18 +13,27 @@ import {
 import { ControlValueAccessor, NgControl } from '@angular/forms';
 
 const cb = () => {};
+
 @Component({
   selector: 'md-editable-textfield',
   template: `
     <div
       *ngIf="isEditing"
       class="md-input-group md-editable-textfield__editing"
-      [ngClass]="textWrapperClasses"
+      [ngClass]="[
+        inputSize ? this.inputSize + ' columns' : '',
```

Comment:
You can only use columns to set set the size?

Reply:
Currently on the react input component, that is a prop we have to do it and translated it to angular. There is also a wrapper class that you can add for custom css.